mockserver.js

```diff
@@ -282,28 +272,84 @@ function getBody(req, callback) {
   });
 }
 
+function isJsonString(str) {
+  if (typeof str !== 'string') {
+    return false;
+  }
+  try {
+    JSON.parse(str);
+  } catch (err) {
+    return false;
+  }
+  return true;
+}
+
+function getMatchingJsonFile(files, fullPath, jsonBody) {
+  for (var file of files) {
+    if (file.endsWith('.json')) {
+      var data = fs.readFileSync(join(fullPath, file), { encoding: 'utf8' });
+
+      try {
+        if (jsonBody === JSON.stringify(JSON.parse(data))) {
```

Comment:
I think you could change this check to `if (JSON.stringify(JSON.parse(jsonBody)) === JSON.stringify(JSON.parse(data))) {` to avoid a `falsy` result due to an extra white space or some other formatting detail.

Reply:
I agree with that because without it, it couldn't work correctly (if some space character exist for example)